appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -84,7 +85,7 @@ internal fun Modifier.styleBorder(
     style: ComponentStyle,
     isDark: Boolean
 ) = this.then(
-    if (style.borderWidth != null && style.borderWidth != 0 && style.borderColor != null) {
+    if (style.borderWidth != null && style.borderWidth != 0.0 && style.borderColor != null) {
```

Comment:
thinking we need to introduce some more robust value comparison for floating point here, rather than using `== 0.0` (and others below)

Reply:
need to see if there is a way to better do this comparison